app/controllers/albums_controller.rb

```diff
@@ -0,0 +1,59 @@
+class AlbumsController < ApplicationController
+
+  def index
+    @albums = Album.all
+  end
+
+  def upvote
+    @album = Album.find(params[:id])
```

Comment:
Because we're setting the `@album` variable in the same with in multiple controller actions (`upvote`, `show`, `edit`, etc.) we should make this a `before_action` which happens on those specific actions.